platform/android/java_godot_wrapper.h

```diff
@@ -34,6 +34,7 @@
 #include "java_godot_view_wrapper.h"
 #include "string_android.h"
 
+#include "core/math/color.h"
```

Comment:
```suggestion
```
Already included

Reply:
Where exactly? I was getting a 'unknown type name 'Color'' error during compilation, which is why I included it.

Reply:
Okay! Thought this included `Object` or `Variant`, then this was not correct!